test_utils/create_validation_csv_files.py

```diff
@@ -14,13 +14,17 @@
 from prepline_sec_filings.fetch import archive_url
 from prepline_sec_filings.sections import SECTIONS_10K, SECTIONS_10Q, SECTIONS_S1, SECSection
 from prepline_sec_filings.sec_document import SECDocument
+from unstructured_api_tools.pipelines.api_conventions import PipelineConfig
+
+
+config = PipelineConfig(os.path.join("..", "preprocessing-pipeline-family.yaml"))
 
 SEC_DOCS_DIR = os.environ.get("SEC_DOCS_DIR")
 CSV_FILES_DIR = os.environ.get("CSV_FILES_DIR")
 FILINGS_MANIFEST_JSON = os.environ.get("FILINGS_MANIFEST_JSON",
                                        os.path.join(SEC_DOCS_DIR, "sec_docs_manifest.json"))
 PIPELINE_SECTION_API_URL = os.environ.get("PIPELINE_SECTION_API_URL",
-                                          "http://127.0.0.1:8000/sec-filings/v0.0.1/section")
+                                          f"http://127.0.0.1:8000/sec-filings/v{config['version']}/section")
```

Comment:
This works, but recommend the simpler `get_pipeline_path("section")` for the route like https://github.com/Unstructured-IO/pipeline-sec-filings/blob/cbbfce71/test_sec_filings/sec_filings/test_section_api.py#L10 and let the method worry about yaml fliles.

Reply:
Done.